documentation/modules/ROOT/pages/connectors/spanner.adoc

```diff
@@ -0,0 +1,1201 @@
+|`long`
+|The number of milliseconds between the last change event's timestamp and the connector processing it.
+The values will incoporate any differences between the clocks on the machines where the database server and the connector are running.
+
+|[[connectors-strm-metric-numberofcommittedtransactions]]<<connectors-strm-metric-numberofcommittedtransactions, `+NumberOfCommittedTransactions+`>>
+|`long`
+|The number of processed transactions that were committed.
+
+|[[connectors-strm-metric-maxqueuesizeinbytes]]<<connectors-strm-metric-maxqueuesizeinbytes, `+MaxQueueSizeInBytes+`>>
+|`long`
+|The maximum buffer of the queue in bytes used to pass events between the streamer and the main Kafka Connect loop.
+
+|[[connectors-strm-metric-currentqueuesizeinbytes]]<<connectors-strm-metric-currentqueuesizeinbytes, `+CurrentQueueSizeInBytes+`>>
+|`long`
+|The current buffer of the queue in bytes used to pass events between the streamer and the main Kafka Connect loop.
+
+|[[connectors-strm-metric-lowwatermark]]<<connectors-strm-metric-lowwatermark, `+LowWatermark+`>>
+|`long`
+|The current low watermark of the connector task. The low watermark describes the time T at which the connector is guaranteed to have streamed out all events with timestamp < T.
+
+|[[connectors-strm-metric-latencylowwatermarklag]]<<connectors-strm-metric-latencylowwatermarklag, `+LatencyLowWatermarkLag+`>>
+|`long`
+|The lag of the low watermark behind the current time in milliseconds. P50, P95, P99, Average, Min, Max metrics are provided.
+
+|[[connectors-strm-metric-latencyspanner]]<<connectors-strm-metric-latencyspanner, `+LatencySpanner+`>>
+|`long`
+|The Spanner commit timestamp to connector read latency. P50, P95, P99, Average, Min, Max metrics are provided.
+
+|[[connectors-strm-metric-latencyreadtoemit]]<<connectors-strm-metric-latencyreadtoemit, `+LatencyReadToEmit+`>>
+|`long`
+|The Spanner read timestamp to connector emit latency. P50, P95, P99, Average, Min, Max metrics are provided.
+
+|[[connectors-strm-metric-latencycommittoemit]]<<connectors-strm-metric-latencycommittoemit, `+LatencyCommitToEmit+`>>
+|`long`
+|The Spanner commit timestamp to connector emit latency. P50, P95, P99, Average, Min, Max metrics are provided.
+
+|[[connectors-strm-metric-latencycommittopublish]]<<connectors-strm-metric-latencycommittopublish, `+LatencyCommitToPublish+`>>
+|`long`
+|The Spanner commit timestamp to Kafka publish timestamp latency. P50, P95, P99, Average, Min, Max metrics are provided.
+
+|[[connectors-strm-metric-latencyemittopublish]]<<connectors-strm-metric-latencyemittopublish, `+LatencyEmitToPublish+`>>
+|`long`
+|The connector emit timestamp to Kafka publish timestamp latency. P50, P95, P99, Average, Min, Max metrics are provided.
+
+|[[connectors-strm-metric-spannereventqueuecapacity]]<<connectors-strm-metric-spannereventqueuecapacity, `+SpannerEventQueueCapacity+`>>
+|`long`
+|The total Spanner event queue capacity.
```

Comment:
Explanation for Spanner event queue and how it differes from the regular above is missing.